pom.xml

```diff
@@ -20,57 +20,38 @@
     </distributionManagement>
 
     <properties>
-        <aem.version>6.4.2</aem.version>
-        <sling.version>1.2.2</sling.version>
-        <geronimo.version>1.0</geronimo.version>
+        <io.wcm.aem.version>6.4.2.0000</io.wcm.aem.version>
+        <io.wcm.aem.mock.version>2.5.0</io.wcm.aem.mock.version>
         <acs-commons.version>3.19.0</acs-commons.version>
-        <servlet.api.version>3.1.0</servlet.api.version>
-        <jsp.api.version>2.1</jsp.api.version>
-        <jcr.version>2.0</jcr.version>
         <taglib.version>5.7.4</taglib.version>
-        <guava.version>22.0</guava.version>
```

Comment:
We'll still need this. Guava 15 won't work for some of our projects, which is what wcm-io is bringing in.

Reply:
Done, I added the Guava 22